### PR TITLE
fix: amend date of certification for verifying certificate

### PR DESCRIPTION
**Pull request by Nil20**
Farajaland PR: https://github.com/opencrvs/opencrvs-countryconfig/pull/150

### CODE REVIEW

**Comment by Nil20** [comment]
Fix: https://github.com/opencrvs/opencrvs-core/pull/7098

File: packages/client/src/views/VerifyCertificate/VerifyCertificatePage.tsx [hunk 1]
```diff
@@ -477,12 +477,14 @@ export function VerifyCertificatePage() {
...
-                          {intl.formatMessage(messageToDefine.createdAt)}
+                          {intl.formatMessage(
+                            messageToDefine.dateOfCertification
+                          )}
                         </Text>
                       }
                       value={
                         <Text variant={'reg16'} element={'span'}>
-                          {formatDate(new Date(data.createdAt), 'dd MMMM yyyy')}
+                          {formatDate(new Date(), 'dd MMMM yyyy')}
```
Comment (naftis) [inline packages/client/src/views/VerifyCertificate/VerifyCertificatePage.tsx:487]: isn't this the page where you land after scanning the QR code on the certificate? so the certification can have happened in the past and isn't necessarily the current date

**Review by naftis — CHANGES_REQUESTED** [review]

File: packages/client/src/views/VerifyCertificate/VerifyCertificatePage.tsx (hunk 1 above)
Comment (Nil20) [reply packages/client/src/views/VerifyCertificate/VerifyCertificatePage.tsx:487]: Yeah. We have to put the last certified date here.

Comment (Nil20) [reply packages/client/src/views/VerifyCertificate/VerifyCertificatePage.tsx]: Updated.

File: packages/client/src/i18n/messages/views/verifyCertificate.ts [hunk 2]
```diff
@@ -110,7 +110,7 @@ const messagesToDefine: IVerifyCertificateMessages = {
     defaultMessage: 'Name of registar',
     description: 'Label for name of registar center'
   },
-  createdAt: {
+  certifiedAt: {
     id: 'verifyCertificate.createdAt',
```
Comment (Zangetsu101) [inline packages/client/src/i18n/messages/views/verifyCertificate.ts:114]:
```suggestion
    id: 'verifyCertificate.certifiedAt',
```
This should prevent confusion when only looking at the translation csv

File: packages/client/src/views/VerifyCertificate/VerifyCertificatePage.tsx [hunk 3]
```diff
@@ -246,6 +247,21 @@ export function VerifyCertificatePage() {
...
+      data.history.find(
+        (h, index, allHistory) =>
+          index - 1 &&
+          allHistory[index]?.regStatus?.toString() ===
+            SUBMISSION_STATUS.CERTIFIED &&
+          allHistory[index + 1]?.regStatus?.toString() !==
+            SUBMISSION_STATUS.CERTIFIED
+      )
+    )?.date
+  }
```
Comment (Zangetsu101) [inline packages/client/src/views/VerifyCertificate/VerifyCertificatePage.tsx:263]: This assumes the tasks are in ascending order according to lastModified time. Then shouldn't just looking for the first "CERTIFIED" item suffice?

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

File: packages/client/src/views/VerifyCertificate/VerifyCertificatePage.tsx (hunk 3 above)
Comment (Nil20) [reply packages/client/src/views/VerifyCertificate/VerifyCertificatePage.tsx:263]: Absolutely! No need to check the status change

**Comment by Zangetsu101** [comment]
Don't we require to create countryconfig PR for the translation change?

**Comment by Nil20** [comment]
> Don't we require to create countryconfig PR for the translation change?

Opened one for the Farajaland. Also need to create one for country config as well, right?

**Comment by Zangetsu101** [comment]
Only needed in countryconfig actually as it will eventually get merged into farajaland

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**